.github/ISSUE_TEMPLATE/Bug_Report.yaml

```diff
@@ -0,0 +1,89 @@
+---
+name: "Bug Report"
+description: Report a bug on Windows Subsystem for Linux
+title: ''
+labels: ''
+assignees: ''
+issue_body: true
+body:
+- type: markdown
+  attributes:
+    value: |
+      Please [search for existing issues](https://github.com/microsoft/WSL/issues) before creating a new one.
+      If you'd like to create a new bug report, please [read the guidance here](https://github.com/Microsoft/WSL/blob/master/CONTRIBUTING.md) before getting started. This will save you time.
+      
+- type: input
+  attributes:
+    label: Windows Build Number
+    description: |
+      Please run `cmd.exe /c ver` to get the build of Windows you are on.
+    placeholder: "Microsoft Windows [Version 10.0.19042.867]"
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: WSL Version
+    description: |
+      Tell us whether the issue is on WSL 2 and/or WSL 1; you can do this by running `cat /proc/version` to check the kernel version. 

```

Comment:
I'd recommend that we change this to: "You can tell your WSL version by running `wsl.exe -l -v`.

Reply:
Great suggestion, fixed!